### ISSUE TITLE
IapExample doesn't catch products

### ISSUE THREAD

**Issue by dguard** [dooboolab/react-native-iap#1653]
> Please use [Discussion board](https://github.com/dooboolab/react-native-iap/discussions) if you want get some help out of it. Please report issue if there is a bug.

I wanted to run IapExample

### Version of react-native-iap
7.5.1

### Version of react-native
0.65.1

### Platforms you faced the error (IOS or Android or both?)
Android

### Expected behavior
![image](https://user-images.githubusercontent.com/1911516/155840124-2970b6fb-517c-4808-b322-1cca007b1346.png)

### Actual behavior
<img src="https://user-images.githubusercontent.com/1911516/155840129-562efb89-4df4-4560-b90a-926b9a9729da.png" width=400>

### Tested environment (Emulator? Real Device?)
Real Device

### Steps to reproduce the behavior
1. git clone https://github.com/dooboolab/react-native-iap
2. cd react-native-iap && yarn
3. cd IapExample && yarn
4. yarn start (first terminal)
5. cd android && ./gradlew bundleAmazonRelease
6. cd .. && react-native run-android --variant=amazonRelease

**Comment by andresesfm** [comment]
I don't think the IapExample app could work out of the box for the following reason:
If you want to test with your own app. Package name (App Id) would have to be set up and match the skus provided.
You would need to use the IAP tester: https://developer.amazon.com/docs/in-app-purchasing/iap-testing-overview.html#receipt-verification-service

**Comment by dguard** [comment]
@andresesfm 
thanks

1. first time it worked with test.sub1 for android
2. then I made a release from version of IapExample directory with same signing keys like my app has
```
cd IapExample
cd android
./gradlew bundleGooglePlayRelease
```
4. uploaded new bundle - it allowed me to add own subscriptions (and products) for the application
5. I added new subscription - it became work via 1 day after adding
6. then I added package react-native-iap for my main package - same like documentation says
7. and it works in main package

Cool !
